.harness/default_input_set.yaml

```diff
@@ -0,0 +1,14 @@
+inputSet:
+  name: default_input_set
```

Comment:
Does this have to be its own file or can it be included in the file it is associated with? Would be easier to manage if it could.